packages/app/client/src/ui/editor/emulator/emulator.tsx

```diff
@@ -360,18 +374,17 @@ export class EmulatorComponent extends React.Component<EmulatorProps, {}> {
         this.props.trackEvent('conversation_restart', {
           userId: 'new',
         });
-        const newUserId = uniqueIdv4();
-        // set new user as current on emulator facilities side
-        await CommandServiceImpl.remoteCall(SharedConstants.Commands.Emulator.SetCurrentUser, newUserId);
-        this.props.updateChat(this.props.documentId, { userId: newUserId });
+        // start conversation with new convo id & user id
+        this.startNewConversation(undefined, true, true);
         break;
       }
 
       case SameUserId:
         this.props.trackEvent('conversation_restart', {
           userId: 'same',
         });
-        this.startNewConversation();
+        // start conversation with new convo id
+        this.startNewConversation(undefined, true);
```

Comment:
Or should it be like this?

```suggestion
        this.startNewConversation(undefined, true, false);
```